### ISSUE TITLE
Not transfering from Windows Chrome/Edge to IOS Safari

### ISSUE THREAD

**Issue by polpo85** [RobinLinus/snapdrop#249]
Transfering from IOS Safari is not working 100% of times but it works. Other way, from Windows to Safari never works in my case. I'm on the same network and accepting Cookies. What can I do? Thank you.

**Comment by BraydenTW** [comment]
Strange. It's working for me. I use Windows Chrome and ios Safari. What are you versions for those two browsers? What OS?

**Comment by polpo85** [comment]
I'm using OS X 14.2 and (Windows) Google Chrome 88.0.4324.104 (Build oficial) (64 bits).

**Comment by BraydenTW** [comment]
Is the device just not showing up? Or not transferring the files?

**Comment by polpo85** [comment]
Not transfering ;(  De: Brayden WEnviado: lunes, 25 de enero de 2021 18:36Para: RobinLinus/snapdropCC: polpo85; AuthorAsunto: Re: [RobinLinus/snapdrop] Not transfering from Windows Chrome/Edge to IOS Safari (#249) Is the device just not showing up? Or not transferring the files?—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by BraydenTW** [comment]
Strange. What type of file are you uploading? Is it too large?

**Comment by polpo85** [comment]
No.. just some images to try. Both devices are connected to same 5G wifi
network. Could affect the 5G?

**Comment by BraydenTW** [comment]
I'm not sure. Do you have devices not connected to a 5G network? Try giving that that a go.

**Comment by polpo85** [comment]
I’ve tried connecting both to normal wifi. Nothing happens when
transferring files between them 🤷🏿‍♀️

**Comment by BraydenTW** [comment]
Not sure then. I would just bring the creator @RobinLinus over here to see if he can help. 

Best of luck!

**Comment by Dre098** [comment]
I’m having the same issue. I haven’t tried photos/videos or other documents/files but I’m trying to send a link by dragging and dropping but it’s not working. Neither is the “send message/text” function when I type something or I copy & paste the link there. Won’t work either way. The device are showing up on both ends but it’s just that the link & text isn’t sending or receiving.

**Comment by theKHR3b** [comment]
Same as @Dre09Dr, showing but not sended, both ways. Tried to send photos.
